challenge-sizebay/src/hooks/useForm.jsx

```diff
@@ -0,0 +1,18 @@
+import {useState} from 'react';
+
+const useForm = (initialState) =>{
+    const [form, setForm] = useState(initialState)
+    
+    const handleInputChange = (event) => {
+        const {value, name} = event.target
```

Comment:
nit: brackets should have a space between their content.

Reply:
This is the comment I mentioned above.

Reply:
@luizfbarbosa12 got it explained at https://github.com/sizebay/frontend-challenge/pull/18#discussion_r1087122398